packages/is/src/index.js

```diff
@@ -6,7 +6,7 @@ export const func = f => typeof f === 'function'
 export const number = n => typeof n === 'number'
 export const string = s => typeof s === 'string'
 export const array = Array.isArray
-export const object = obj => obj && !array(obj) && typeof obj === 'object'
+export const object = obj => obj && !array(obj) && typeof obj === 'object' && obj !== null
```

Comment:
that's already handled by `obj &&` part, it wont proceed further when `obj === null`

Reply:
Yeah I got it. Thanks. I didn't look at the first part. My bad.